src/library/basetrackcache.cpp

```diff
@@ -500,6 +500,11 @@ void BaseTrackCache::filterAndSort(const QSet<TrackId>& trackIds,
     if (!filter.isEmpty()) {
         filter.prepend("WHERE ");
     }
+    qDebug() << "   .";
+    qDebug() << "   .";
+    qDebug() << "   query filter:" << filter;
+    qDebug() << "   .";
+    qDebug() << "   .";
```

Comment:
Interesting I use qDebug() < "##############################"; 
to find my debug. 
Maybe we can build a wrapper for this ...

Reply:
yeah, I'm tired of using this since it also needs too much space in the editor. the blank lines indentation are really helpful to spot it quickly.